site/content/en/docs/faq/_index.md

```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How to create an multiple clusters using minikube ?
+
+By default `minikube start` creates a cluster named minikube, if you would like to create another cluster or change the name.
```

Comment:
By default, `minikube start` creates a cluster named `minikube`. If you would like to create a different cluster or change its name, you can use the `--profile` (or `-p`) flag, which will create a cluster with the specified name.